### PR TITLE
Docs - Record identifiers page, fixes 526

### PR DESCRIPTION
**Pull request by kd-ods**
# Overview

- What does this pull request do?

Adds new page content, including 3 images.

- How can a reviewer test or examine your changes?

Build the docs and read the page.

- Who is best placed to review it?

@kathryn-ods 

(Closes/Relates to) issue: #526

## Translations

- [x] New or edited strings appearing as a result of this PR will be picked up for translation
- [x] N/A - I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [x] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [x] - N/A I've updated the changelog, if necessary.
- [x] - N/A I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [x] - N/A I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

File: docs/schema/guidance/record-ids.rst [hunk 1]
```diff
@@ -4,4 +4,53 @@
 Record identifiers
 ==================
 
-[Stub page]
+To publish beneficial ownership data in BODS format, an information management system MUST maintain record identifiers for each element (person, entity and relationship) in each beneficial ownership network.
+
+Record identifiers MUST be unique within the publisher's system and never reused to refer to a different element. 
+
+There are situations where a publisher's system can allow information about the same person, entity or relationship to be collected in relation to the beneficial ownership networks of two (or more) different subjects. In these cases, information from different sources about the same person, entity or relationship can exist side-by-side and needs to be represented in published data. In these cases, each set of information MUST have its own record identifier. (See the example below.)
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:11]: I'm finding this a bit difficult to understand - I think because it's talking both about overlapping networks and different sources. Is the multiple sources the key aspect here? If so maybe reword along these lines?

'Information about one element MAY be submitted by multiple sources. Each set of information about an element from a different source MUST have its own record identifier. (See the example below.)'

File: docs/schema/guidance/record-ids.rst [hunk 2]
```diff
@@ -4,4 +4,53 @@
...
+
+The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person in question, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.  
+
+The identity of a relationship is constituted by the identities of its subject and interested party. Therefore the value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:15]: I think this lead in sentence can be deleted. 
```suggestion
The value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
```

File: docs/schema/guidance/record-ids.rst [hunk 3]
```diff
@@ -4,4 +4,53 @@
...
 
-[Stub page]
+To publish beneficial ownership data in BODS format, an information management system MUST maintain record identifiers for each element (person, entity and relationship) in each beneficial ownership network.
+
+Record identifiers MUST be unique within the publisher's system and never reused to refer to a different element. 
+
+There are situations where a publisher's system can allow information about the same person, entity or relationship to be collected in relation to the beneficial ownership networks of two (or more) different subjects. In these cases, information from different sources about the same person, entity or relationship can exist side-by-side and needs to be represented in published data. In these cases, each set of information MUST have its own record identifier. (See the example below.)
+
+The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person in question, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:13]:
```suggestion
The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.  
```

File: docs/schema/guidance/record-ids.rst [hunk 4]
```diff
@@ -4,4 +4,53 @@
...
+The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person in question, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.  
+
+The identity of a relationship is constituted by the identities of its subject and interested party. Therefore the value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
+
+Record identifiers are used for :ref:`representing-bo` and for :ref:`guidance-updating-data`.
+
+Example scenario
+----------------
+
+Private companies in a given jurisdiction are obliged to disclose to the Company Registration Authority: their legal owners and any beneficial owners with direct interests in the company.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:22]:
```suggestion
Private companies in a jurisdiction are obliged to disclose to the Company Registration Authority: their legal owners and any beneficial owners with direct interests in the company. 
```

Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:22]:
```suggestion
Private companies in a given jurisdiction are obliged to disclose their legal owners to the Company Registration Authority.
```

Is the distinction between a direct BO and a legal owner adding anything to the example?

File: docs/schema/guidance/record-ids.rst [hunk 5]
```diff
@@ -4,4 +4,53 @@
...
+The identity of a relationship is constituted by the identities of its subject and interested party. Therefore the value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
+
+Record identifiers are used for :ref:`representing-bo` and for :ref:`guidance-updating-data`.
+
+Example scenario
+----------------
+
+Private companies in a given jurisdiction are obliged to disclose to the Company Registration Authority: their legal owners and any beneficial owners with direct interests in the company. 
+
+In the declaration from Alpha Corp., it discloses one natural person as a beneficial owner (and legal owner) and it also discloses Beta Systems as a legal owner.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:24]: There's a mix of passive/active voice here. 

Maybe 'Apha Corp. declares a natural person and Beta systems as its legal owners' or 'Alpha Corp. makes a declaration disclosing...' (also depends on how important distinction of legal and beneficial owners is)

File: docs/schema/guidance/record-ids.rst [hunk 6]
```diff
@@ -4,4 +4,53 @@
...
+
+.. figure:: ../../_assets/AlphaCorp.svg
+   :alt: Diagram of a beneficial ownership network. An entity node for the company Alpha Corp is connected via a solid line to a person node. Alpha Corp's node is also connected via a solid line to an entity node for the company Beta Systems. An entity statement block sits next to Alpha Corp's node. Within the entity statement, recordId is '04a221be'. An entity statement block sits next to Beta Systems's node. Within the entity statement, recordId is '09d3a4fb'. 
+   :figwidth: 80%
+   :align: center
+
+In the declaration from Beta Systems, it declares that it has four direct beneficial owners.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:31]:
```suggestion
Beta Systems declares that it has four direct beneficial owners. 
```

**Review by kathryn-ods — COMMENTED** [review]
Generally all looking good - I've added some suggestions for readability

File: docs/schema/guidance/record-ids.rst (hunk 1 above)
Comment (kd-ods) [reply docs/schema/guidance/record-ids.rst:11]: > Is the multiple sources the key aspect here?

It's really the overlapping networks (in the sense of them containing the same element) that is the key. I'm going to remove the reference to 'sources' since that's a bit of a red herring.

File: docs/schema/guidance/record-ids.rst (hunk 2 above)
Comment (kd-ods) [reply docs/schema/guidance/record-ids.rst:15]: I'm going to leave this in since it's a foundational principle that determines when a relationship record should and shouldn't be updated.

File: docs/schema/guidance/record-ids.rst (hunk 4 above)
Comment (kd-ods) [reply docs/schema/guidance/record-ids.rst:22]: I'm going to leave it in, since it retains the BO link. (If we only talk about legal ownership, that might prompt unnecessary questions.)

File: docs/schema/guidance/record-ids.rst (hunk 5 above)
Comment (kd-ods) [reply docs/schema/guidance/record-ids.rst:24]: Yes - will edit.

**Comment by kd-ods** [comment]
This is ready for re-review, @kathryn-ods.

File: docs/schema/guidance/record-ids.rst [hunk 7]
```diff
@@ -4,4 +4,53 @@
...
+When the BODS data is published, there are statements about the two new Beta Systems records, as below. 
+
+.. figure:: ../../_assets/AlphaBeta.svg
+   :alt: Diagram of two entity statement blocks. The first has: recordId '09d3a4fb', recordDetails.name 'BetaSys', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '04a221be'. The second has: recordId '02c83aa2', recordDetails.name 'Beta Systems', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '02c83aa2' 
+   :figwidth: 100%
+   :align: center
+
+Note that:
+
+- The name of the company has not been consistently disclosed.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:49]:
```suggestion
- the name of the company has not been consistently disclosed. 
```
small style guide edit

File: docs/schema/guidance/record-ids.rst [hunk 8]
```diff
@@ -4,4 +4,53 @@
...
+
+.. figure:: ../../_assets/AlphaBeta.svg
+   :alt: Diagram of two entity statement blocks. The first has: recordId '09d3a4fb', recordDetails.name 'BetaSys', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '04a221be'. The second has: recordId '02c83aa2', recordDetails.name 'Beta Systems', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '02c83aa2' 
+   :figwidth: 100%
+   :align: center
+
+Note that:
+
+- The name of the company has not been consistently disclosed. 
+- The two statements do pertain to the same real world entity, as the ``scheme`` and ``id`` properties in both statements are the same.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:50]:
```suggestion
- the two statements do pertain to the same real world entity, as the ``scheme`` and ``id`` properties in both statements are the same.
```

File: docs/schema/guidance/record-ids.rst [hunk 9]
```diff
@@ -4,4 +4,53 @@
...
+- The two statements about the same real world entity relate to two different declaring subjects, as shown by the different ``declarationSubject`` values.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:51]:
```suggestion
- the two statements about the same real world entity relate to two different declaring subjects, as shown by the different ``declarationSubject`` values.
```

File: docs/schema/guidance/record-ids.rst [hunk 10]
```diff
@@ -4,4 +4,53 @@
...
+- The statement for ‘Beta Systems’ has been disclosed by Beta Systems itself, since the ``recordId`` value is also given as the ``declarationSubject``.
```
Comment (kathryn-ods) [inline docs/schema/guidance/record-ids.rst:52]:
```suggestion
- the statement for ‘Beta Systems’ has been disclosed by Beta Systems itself, since the ``recordId`` value is also given as the ``declarationSubject``.
```

**Review by kathryn-ods — APPROVED** [review]
approved - tiny style guide edit

**Review by kathryn-ods — APPROVED** [review]
